### PR TITLE
feat(dataframe): support synchronizing operations of the Series

### PR DESCRIPTION
**Pull request by graczhual**

### CODE REVIEW

**Comment by GravitiRob** [comment]
Task linked: [CU-2rbp7qj 支持将 DataFrame 成员的行操作同步回主 DataFrame](https://app.clickup.com/t/2rbp7qj)

File: graviti/dataframe/column/series.py [hunk 1]
```diff
@@ -136,6 +137,18 @@ def __setitem__(
             raise TypeError("The schema of the given Series is mismatched")
 
         self._set_item_by_slice(key, value)
+        if self.operations._data is not None:
+            _record_key = self._record_key
+            record_key = [_record_key[i] for i in range(len(value) + 1)[key]]  # type: ignore[index]
```
Comment (AChenQ) [inline graviti/dataframe/column/series.py:142]: How about
`[_record_key[i] for i in range(*key.indices(key.stop))`

File: graviti/dataframe/container.py [hunk 2]
```diff
@@ -9,36 +9,67 @@
...
     from graviti.dataframe.frame import DataFrame
 
 _T = TypeVar("_T", bound="Container")
 
 
+class Operations(UserMutableSequence[DataFrameOperation]):
+    """The basic structure of the operations of the DataFrame.
+
+    Arguments:
+        is_sheet: Whether the DataFrame to which the operations belongs is a sheet of the draft.
```
Comment (AChenQ) [inline graviti/dataframe/container.py:28]: wrong docstring

**Review by AChenQ — CHANGES_REQUESTED** [review]

File: graviti/dataframe/column/series.py [hunk 3]
```diff
@@ -383,7 +396,7 @@ class Series(SeriesBase):  # pylint: disable=abstract-method
         0 1
         1 2
         2 3
-        3 4
+    3 4
```
Comment (AChenQ) [inline graviti/dataframe/column/series.py:399]: align

**Review by linjiX — APPROVED** [review]

**Review by AChenQ — APPROVED** [review]

**Pull request merged**